### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by ferricoxide**
With the recent spate of issue-opens, probably good to add some more project-documentation.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - VirtualBox image in [Vagrant Cloud](https://vagrantcloud.com/)
+    - VMware image in Vagrant Cloud<sup>1</sup>
+- Proliferations for each of the above repositories exist for
+    - Red Hat 6
+    - CentOS 6
+    - Red Hat 7<sup>2</sup>
+    - CentOS 7<sup>2</sup>
+- Images are produced monthly. This means maintaining 28 images per month for a minimum time-span of six to twelve months.
+
+Additionally, the STIG contents contain multiple scanning/hardening profiles. To support each profile would require a unique, pre-hardened image for each öff the shelf" profile. This does not account for custom scanning/hardening profiles. Supporting _all_ of the "off the shelf" profiles via pre-hardened images would require generating 100+ images per month. Not practical on a monthly basis; even less practical when extended across the six- to twelve-month lifespan of images in multiple deployment domains (i.e., AWS, Vagrant Cloud ...and eventually Azure and possibly others).
```
Comment (lorengordon) [inline docs/FAQ.md]: `öff`?

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Because of the above, we opted to keep AMIs as minimally-hardened as possible - instead choosing to apply hardenings at launch-time using other frameworks.
+
+### Q: So... Why would I use these images, then?
+
+In general, once an image is launched as a VM, it requires considerable gymnastics to re-layout the storage to meet STIG requirements. Those gymnastics can vary from simply annoyingly labor-intensive to effectively "not possible". This set of images solves that problem
```
Comment (lorengordon) [inline docs/FAQ.md]: Period to close the paragraph...

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+A. This FAQ is for using spel. That said Watchmaker includes a full [documentation set](https://watchmaker.readthedocs.io) that should help you with its use.
+
+
+### Q. My application won't work under FIPS: now what?
+
+A. If you're using EL6 images, this is a non-problem. If you're using EL7, things are a bit more challenging. Our images are FIPS-enabled because the STIGs say they need to be. As such, our users ultimately need to figure out how to get their app to work under FIPS or get an exception from their security team (sorta like firewalld and SELinux - also baked in to the EL7 images). These images are meant as a 90% solution. If you're one of the unlucky 10% whose app won't work under FIPS in EL7, the best we can suggest is to let your provisioning framework handle the problem for you.
+
+### Q. But I'm following your suggestion to use Watchmaker: can that help me with toggling FIPS mode?
+
+A. Yes. See watchmaker's documentation for guidance.
```
Comment (lorengordon) [inline docs/FAQ.md]: Maybe link right to the watchmaker FAQ here, https://watchmaker.readthedocs.io/en/stable/faq.html? That's where we'll add this guidance.

File: docs/FAQ.md (hunk 2 above)
Comment (ferricoxide) [reply docs/FAQ.md]: Done.

File: docs/FAQ.md (hunk 3 above)
Comment (ferricoxide) [reply docs/FAQ.md]: Done

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+A. This FAQ is for using spel. That said Watchmaker includes a full [documentation set](https://watchmaker.readthedocs.io) that should help you with its use.
+
+
+### Q. My application won't work under FIPS: now what?
+
+A. If you're using EL6 images, this is a non-problem. If you're using EL7, things are a bit more challenging. Our images are FIPS-enabled because the STIGs say they need to be. As such, our users ultimately need to figure out how to get their app to work under FIPS or get an exception from their security team (sorta like firewalld and SELinux - also baked in to the EL7 images). These images are meant as a 90% solution. If you're one of the unlucky 10% whose app won't work under FIPS in EL7, the best we can suggest is to let your provisioning framework handle the problem for you.
+
+### Q. But I'm following your suggestion to use Watchmaker: can that help me with toggling FIPS mode?
+
+A. Yes. See watchmaker's [documentation](https://watchmaker.readthedocs.io/en/stable/faq.html?) for guidance.
```
Comment (lorengordon) [inline docs/FAQ.md]: Rando question mark in the url...

Comment (ferricoxide) [reply docs/FAQ.md]: Lulz... Copy-paystah

Comment (ferricoxide) [reply docs/FAQ.md]: Gone

File: README.md [hunk 5]
```diff
@@ -1,15 +1,22 @@
...
+
+VMs' root filesystems are generally not live-repartitionable once launced from their images. As a result, if a STIG-scan is performed against most of the community-published images for Red Hat and CentOS, those scans will note failures for each of the various "`${DIRECTORY}` is on its own filesystem" tests. The images produced through this project are designed to ensure that these particular scan-failures do not occur. 
+
+Aside from addressing the previously-noted partitioning findings, spel does _not_ apply any STIG-related hardening. The spel-produced images are expected to act as a better starting-point in a larger hardening process.
+
+If your organization does not already have an automated hardening process, please see our tool, [Watchmaker](https://github.com/plus3it/watchmaker.git). This tool is meant to help spel-users (and users of other Enterprise Linux images) by performing launch-time hardening activities.
+
+## We have a FAQ now!
+
+We've added an [FAQ](doct/FAQ.md) to the project. Hopefully, your questions are answered there. If they aren't, please feel free to submit an issue requesting an appropriate FAQ entry.
```
Comment (lorengordon) [inline README.md]: `doct`, bad path?

Comment (ferricoxide) [reply README.md]: can't event claim fat-fingers on that one. I mean, I could, but given the s-to-t distance, I'd have to be typing with hooves.

Fixed and pushed

**Pull request merged**